client/coral-framework/components/CreateDisplayNameDialog.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
These components probably make more sense to live in the `coral-plugin-sign-in` folder since that's where a lot of similar code is from.